app/ViewModels/Concerns/Transaction/InteractsWithVotes.php

```diff
@@ -17,16 +17,24 @@ public function voted(): ?WalletViewModel
         }
 
         $payload = $this->rawPayload();
+
+        // @codeCoverageIgnoreStart
+        // It is possible to reach this point if `rawPayload` returns null, which
+        // is theoretically only possible if the `stream_get_contents` function
+        // called in `app/ViewModels/Concerns/Transaction/HasPayload.php@rawPayload`
+        // returns false, something I have not been able to replicate in tests.
```

Comment:
do you think it's worth removing the check in this case? :thinking: we could always re-add it if we experience the problem in the future with a transaction that causes it, then we'll have a transaction's data we can use to test the functionality

Reply:
can ignore that, I checked with mici and we'll keep them :oleglala: